LibreNMS/Validations/Updates.php

```diff
@@ -66,7 +66,7 @@ public function validate(Validator $validator): void
             $remote_ver = Version::get()->remoteCommit();
             if ($local_ver['sha'] != ($remote_ver['sha'] ?? null)) {
                 if (empty($local_ver['date'])) {
-                    $process = new Process(['git', 'show', '--no-patch', '--pretty=%H|%ct'], base_path());
```

Comment:
Oh, this should call `Version::get()->localCommit()`  (or preferrably set $version first)

Reply:
nevermind, it is ran only get the error output :D